docs/sources/administration/roles-and-permissions/access-control/plan-rbac-rollout-strategy/index.md

```diff
@@ -170,63 +170,91 @@ curl --location --request POST '<grafana_url>/api/access-control/roles/' \
 
 By default, only a Grafana Server Admin can create and manage custom roles. If you want your `Editors` to do the same, [update the `Editor` basic role permissions]({{< ref "./manage-rbac-roles.md#update-basic-role-permissions" >}}). There are two ways to achieve this:
 
-- Add the `fixed:roles:writer` role permissions to the `basic:editor` role using the `role > from` list of your provisioning file:
-
-```yaml
-apiVersion: 2
-
-roles:
-  - name: 'basic:editor'
-    global: true
-    version: 3
-    from:
-      - name: 'basic:editor'
-        global: true
-      - name: 'fixed:roles:writer'
-        global: true
-```
-
-- Or add the following permissions to the `basic:editor` role, using provisioning or the [RBAC HTTP API]({{< relref "../../../../developers/http_api/access_control/#update-a-role" >}}):
-
-| action         | scope                       |
-| -------------- | --------------------------- |
-| `roles:read`   | `roles:*`                   |
-| `roles:write`  | `permissions:type:delegate` |
-| `roles:delete` | `permissions:type:delegate` |
+- Add the following permissions to the `basic:editor` role, using provisioning or the [RBAC HTTP API]({{< relref "../../../../developers/http_api/access_control/#update-a-role" >}}):
+
+  | action         | scope                       |
+  | -------------- | --------------------------- |
+  | `roles:read`   | `roles:*`                   |
+  | `roles:write`  | `permissions:type:delegate` |
+  | `roles:delete` | `permissions:type:delegate` |
+
+  As an example, here is a small bash script that fetches the role, modifies it using `jq` and updates it:
+
+  ```bash
+  # Fetch the role, modify it to add the desired permissions and increment its version
```

Comment:
Generally it's better to provide this information outside of the code block so that it can be more easily read by the user.